app/helpers/records_helper.rb

```diff
@@ -33,17 +33,17 @@ def search_tab
   def filter_facets
     # facets that we want to ignore/exclude
     blacklist = ['type']
-    facets = Hash[ @search.facets.map { |facet| [facet.name.to_s, facet.values] }]
+    facets = Hash[@search.facets.map { |facet| [facet.name.to_s, facet.values] }]
     # Remove empty facets & unwanted facets
-    facets = facets.delete_if { |facet, values| values.empty? || blacklist.include?(facet)}
+    facets.delete_if { |facet, values| values.empty? || blacklist.include?(facet) }
   end
 
   def filter_class(filter_name)
-    filter_name.strip.gsub(/[^[:alnum:]]/, "").downcase
+    filter_name.strip.gsub(/[^[:alnum:]]/, '').downcase
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.